blockchain/fundamentals/intro-blockchain/importance-of-blockchain.md

```diff
@@ -0,0 +1,42 @@
+---
+author: mihaiberq
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+links:
+  - '[Examples of Blockchain Use-Cases](https://medium.com/@matteozago50-examples-of-how-blockchains-are-taking-over-the-world-4276bf488a4b){website}'
+
+---
+
+# Why Should You Care?
+
+---
+## Content
+           	
+Here are a couple reasons on why blockchain is an important technology:
+
+1. Secure
+All data found in the blockchain is secured using advanced cryptography. This makes it resistant to unauthorized changes (like hacking).
+
+Because the blockchains is decentralized, it is less susceptible to human error, data loss or hacking.
```

Comment:
```suggestion
Because the blockchain is decentralized, it doesn't have a single point of failure. This makes it less susceptible to abuse, human error, data loss or hacking.
```